source/fundamentals/crud/read-operations/retrieve.txt

```diff
@@ -63,6 +63,49 @@ filter as a ``SingleResult`` type.
 To learn how to access data in a ``SingleResult`` see the :ref:`BSON
 <bson-unmarshalling>` guide.
 
+Modify Behavior
+~~~~~~~~~~~~~~~
+
+The ``Find()`` and ``FindOne()`` functions optionally take a
+``FindOptions`` and ``FindOneOptions`` type, which represents options
```

Comment:
It seems a bit redundant to say "FindOptions" ... "represents options that you can use to modify behavior". They are called "Options" and by definition they modify behavior. Maybe you could get the point across by rearranging the sentence:
>You can modify the behavior of `Find()` and `FindOne()` by passing ....